added homeWork 3

SergeyLysovAPP opened the pull request:
1. Описать несколько структур – любой легковой автомобиль SportCar и любой грузовик TrunkCar.

2. Структуры должны содержать марку авто, год выпуска, объем багажника/кузова, запущен ли двигатель, открыты ли окна, заполненный объем багажника.

3. Описать перечисление с возможными действиями с автомобилем: запустить/заглушить двигатель, открыть/закрыть окна, погрузить/выгрузить из кузова/багажника груз определенного объема.

4. Добавить в структуры метод с одним аргументом типа перечисления, который будет менять свойства структуры в зависимости от действия.

5. Инициализировать несколько экземпляров структур. Применить к ним различные действия.

6. Вывести значения свойств экземпляров в консоль.

SergeyLysovAPP committed:
added homeWork 3

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,216 @@
+    case offline = "остановлен"
+}
+
+enum WindowsStatus: String{
```
```suggestion
enum WindowsStatus: String {
```

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,216 @@
+    let fuelСonsumption: UInt
+    let fuelMaxVolume: UInt
+    
+    var powerReserve: UInt {
```
можно без гет, просто ретерн

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,216 @@
+    
+    var powerReserve: UInt {
+        
+        get {
```
и тут без гет

iMrOwl commented on homeWork/Sources/main.swift:
```diff
@@ -7,5 +7,216 @@
+    
+    mutating func startEngine() {
+        
+        self.engineStatus = .online
```
убрать селф

SergeyLysovAPP committed:
Update homeWork/Sources/main.swift

Co-authored-by: Евгений Сычёв <38351470+iMrOwl@users.noreply.github.com>

SergeyLysovAPP committed:
Сode optimization

- delete ".self" in mutating methods
- delete "get" in properties without "set"
